src/client/java/dev/creesch/model/WebsocketJsonMessage.java

```diff
@@ -0,0 +1,53 @@
+package dev.creesch.model;
+
+import com.google.gson.JsonElement;
+import com.google.gson.JsonParser;
+import com.google.gson.annotations.SerializedName;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.AllArgsConstructor;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+public class WebsocketJsonMessage {
+    private long timestamp;
+    private ChatServerInfo server;
+    private MessageType type;
+    @SerializedName("minecraftVersion")
```

Comment:
Would this normally serialize as `minecraft_version`?

Reply:
Nope, can't even remember why I added the annotation there. Going to remove it.